src/utils/sets-equal.js

```diff
@@ -30,11 +30,10 @@ export default function setsEqual(setA, setB) {
     return false;
   }
 
-  for (const a of setA) {
-    if (!setB.has(a)) {
-      return false;
-    }
-  }
+  let acc = true;
+  setA.forEach((a) => {
+    acc = acc && setB.has(a);
+  });
 
-  return true;
+  return acc;
```

Comment:
I see the PR includes moře commits from my Fork, it should have only the First oné.